.buildkite/pipeline.yml

```diff
@@ -1,55 +0,0 @@
-steps:
```

Comment:
keep buildkite for the moment, we need to build on both for some days to check that everything is ok

Reply:
ok